lib/ansible/plugins/connection/podman.py

```diff
@@ -163,10 +163,18 @@ def put_file(self, in_path, out_path):
         display.vvv("PUT %s TO %s" % (in_path, out_path), host=self._container_id)
         if not self._mount_point:
             rc, stdout, stderr = self._podman(
-                "cp", [in_path, self._container_id + ":" + out_path], use_container_id=False)
+                "cp", [in_path, self._container_id + ":" + out_path], use_container_id=False
+            )
             if rc != 0:
-                raise AnsibleError("Failed to copy file from %s to %s in container %s\n%s" % (
-                    in_path, out_path, self._container_id, stderr))
+                if to_native('cannot copy into running rootless container with pause set' in to_native(stderr)):
```

Comment:
You shouldn't need to use `to_native()` on an already native string. I'm also not sure if `to_native()` was meant to wrap the whole conditional statement. That seems like it would result in `'False'` or `'True'` which is a truthy string.
```suggestion
                if 'cannot copy into running rootless container with pause set' in to_native(stderr):
```

Reply:
I wasn't thinking about wrapping the whole thing for truthy strings, was just on a `to_native()` roll and threw on where we don't need one. Thanks!